src/tribol/interface/tribol.hpp

```diff
@@ -178,6 +179,62 @@ void registerMesh( integer meshId,
                    const real* y,
                    const real* z=nullptr );
 
+/*!
+ * @brief Define and register a coupling scheme over an MFEM mesh.
+ *
+ * This method is designed to enable simple registration of a contact coupling
+ * scheme over a parallel decomposed MFEM mesh.  Contact surfaces are defined
+ * via a list of boundary attributes, which is used to construct a contact
+ * surface mfem::ParSubMesh.  The parallel domains are then rebalanced using the
+ * redecomp library and the redecomp-level mesh and fields are automatically
+ * registered with Tribol.  Mesh re-decomposition is done with each call to
+ * update(), ensuring the redecomp mesh is still representative of the current
+ * state of the mesh.
+ *
+ * @param [in] cs_id Index to use for the coupling scheme.
+ * @param [in] mesh_id_1 The first ID of the contact surface mesh.
+ * @param [in] mesh_id_2 The second ID of the contact surface mesh.
+ * @param [in] mesh MFEM volume mesh.
+ * @param [in] current_coords Coordinates associated with mesh.
```

Comment:
Does this need to be passed in as an argument, or can Tribol grab it from the parmesh?

Reply:
It's unclear whether the coordinates are reference or current in the `mfem::ParMesh`; also the Nodes grid function may not exist for a linear mesh.  Better to explicitly require a coordinates grid function.